.github/workflows/build_and_test.yml

```
@@ -297,6 +298,80 @@ jobs:
           fail_ci_if_error: false
           verbose: true
 
+  macos-m1:
+    runs-on: ${{ matrix.os-version }}
+    strategy:
+      matrix:
+        os-version: [macos-14]
+        arch: [arm64]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v5
+        with:
+          # support version: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
+          python-version: "3.12.x"
+
+      - name: Configure PcapPlusPlus
+        # Ensure user have access to network devices instead of giving super-user right
+        # Seems macos-14 has more strict file permission
+        run: |
+          sudo chmod a+rw /dev/bpf*
+          sudo chmod a+rw /usr/local
+          cmake -DCMAKE_OSX_ARCHITECTURES=${{ matrix.arch }} -DPCAPPP_BUILD_COVERAGE=ON -S . -B "$BUILD_DIR"
+
+      - name: Build PcapPlusPlus
+        run: cmake --build "$BUILD_DIR" -j
+
+      - name: Check architecture
+        run: lipo $BUILD_DIR/Pcap++/libPcap++.a -verify_arch ${{ matrix.arch }}
+
+      - name: Install Tcpreplay
+        run: brew install tcpreplay
+
+      - name: Test PcapPlusPlus
+        run: |
+          python -m pip install -U pip
+          python -m pip install -r ci/run_tests/requirements.txt
+          python ci/run_tests/run_tests.py --interface en0
+
+      - name: Test Examples
+        run: |
+          cd Tests/ExamplesTest
+          python -m pip install -U pip
+          python -m pip install -r requirements.txt
+          python -m pytest --interface en0 --use-sudo --root-path=../../Dist/examples_bin
+
+      - name: Check installation
+        run: |
+          cmake -DPCAPPP_BUILD_COVERAGE=OFF -S . -B "$BUILD_DIR"
+          cmake --build "$BUILD_DIR" -j
+          cmake --install $BUILD_DIR
```

Better to put path between " " in case there is some spaces in the path

```suggestion
          cmake --install "$BUILD_DIR"
```

---

fixed.